.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,60 @@
+name: 🐛 Bug
+description: Report an issue to help improve the project.
+labels: ["bug"]
+body:
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of the question or issue, also include what you tried and what didn't work
+    validations:
+      required: true
+      
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Reproduction steps
+      description: Please enter an explicit description of your issue
+      value: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+      render: bash
+      
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: "Expected behavior"
+      description: Please enter an expected behavior of code
+      
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: Please add screenshots if applicable
+    validations:
+      required: false
+    
+  - type: dropdown
+    id: os
```

Comment:
This project is a web app, which means the environment should also include browser information

e.g. of good environment info
![image](https://user-images.githubusercontent.com/34387770/209744938-428e677f-e27f-4325-b680-467f7d00a54e.png)

Reference:
[how to check Chrome version](https://www.google.com/chrome/update/)